gcloud_bigtable/google/bigtable/admin/cluster/v1/bigtable_cluster_service_api.py

```diff
@@ -0,0 +1,262 @@
+# Copyright 2015 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# EDITING INSTRUCTIONS
+# This file was generated from google/bigtable/admin/cluster/v1/bigtable_cluster_service.proto,
+# and updates to that file get reflected here through a regular refresh
+# process. However, manual and updates to that file get reflected here
+# through a regular refresh additions are allowed because the refresh
+# process performs a 3-way merge in order to preserve those manual additions.
+# In order not to break the refresh process, only certain types of
```

Comment:
FWIW, I'm pretty heavily opposed to *ever* editing any generated module, unless the generation is intended only as a starting point (never to be regenerated).  Application code should just import and use the generated code.